client/components/login/login.test.js

```diff
@@ -219,10 +222,12 @@ describe("<Login /> interactions", () => {
       });
     });
 
+    expect(wrapper.find("#email")).toEqual({});
+    expect(wrapper.state("phone_number")).toEqual("");
     wrapper
-      .find("#email")
-      .simulate("change", { target: { value: "test email", name: "email" } });
-    expect(wrapper.state("email")).toEqual("test email");
+      .find("[name='phone_number']")
+      .simulate("change", {target: {value: "+393660011333", name: "phone_number"}});
+    expect(wrapper.state("phone_number")).not.toEqual("");
```

Comment:
ensure there's 1 test for the email scenario and 1 for the phone number scenario please